route_optimizer/app/storage_manager.py

```
@@ -0,0 +1,152 @@
+        self.routes_prefix = "routes/"
+        self.client = None
+        self.bucket = None
+        
+        if self.bucket_name and GCS_AVAILABLE:
+            try:
+                self.client = gcs.Client()
+                self.bucket = self.client.bucket(self.bucket_name)
+                logger.info(f"GCS接続成功: {self.bucket_name}")
+            except Exception as e:
+                logger.error(f"GCS接続エラー: {e}")
+                st.warning(f"⚠️ GCS接続エラー: ローカルストレージを使用します")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)